### PR TITLE
Added POSITION HOLD flight mode show

### PR DESCRIPTION
**Pull request by demvlad**
The improvement for ["Position hold for 4.6" #13975](https://github.com/betaflight/betaflight/pull/13975) PR
Can use for this PR right now.
But can not merged in master till.
PosHold

### CODE REVIEW

File: src/flightlog_fielddefs.js [hunk 1]
```diff
@@ -143,6 +143,7 @@ export const FLIGHT_LOG_FLIGHT_MODE_NAME_POST_4_5 = makeReadOnly([
   "HEADFREE",
   "PASSTHRU",
   "FAILSAFE",
+  "POSHOLD",
```
Comment (haslinghuis) [inline src/flightlog_fielddefs.js:146]: Think we need to revisit FM's added from 4.5. and add them instead to

https://github.com/betaflight/blackbox-log-viewer/pull/788/files#diff-1c35af2d073b48014296ed8da03ed89c112f28d05456355d2bf04f3fb3e92ce4R538-R540

Comment (demvlad) [reply src/flightlog_fielddefs.js:146]: We have new AltHold and PosHold modes after 4.5. Its are included in modes list now.

Comment (haslinghuis) [reply src/flightlog_fielddefs.js:146]: These flight modes were added in 4.6. Are they backwards compatible using 4.5 logs ?

Comment (demvlad) [reply src/flightlog_fielddefs.js:146]: The new AltaHold and PosHold modes names are added for BF 4.6 FW versions only.
The previouse BF versions use own names list.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**